docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,153 @@
+.. _acsincome-data:
+Retiring Adult: ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset.
+The authors compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. [1]_ filtered the data such that ACSIncome only includes individuals above 16 years old 
+who worked at least 1 hour per week in the past year and had an income of at least $100.
+
+
+.. _acsincome_dataset_description:
+
+Dataset Description
+^^^^^^^^^^^^^^^^^^^
+The authors provide data from 2014-2018 for all 50 states and Puerto Rico.
+We uploaded the 2018 data to OpenML, which fairlearn can access.
```

Comment:
Do we want to eventually support the ACS datasets from the other years (2014-2017) as well? One of the key benefits of these datasets, as listed in the paper by the authors, was to perform temporal evaluation of a fairness criterion and intervention.

Reply:
Great question. Yes I think we want to support these in the future. My opinion was that the scope of this PR should only add 2018 data for one of the datasets (ACSIncome) as a starting point. Ideas for future PRs include 
1) Adding other datasets from the paper, such as ACSPublicCoverage which @britneyting is working on
2) Adding analysis of how the composition of the new ACSIncome dataset compares to UCI Adult. Some interesting questions could include: Are certain groups better represented in the updated dataset? Are any discrepancies larger or smaller? 
3) Adding data from other years